content/03.results.md

```diff
@@ -134,27 +134,27 @@ Following SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G*
 
 <!-- TODO: Update group-specific count numbers below with figure update -->
 #### Low-grade gliomas
-As expected, the majority (62%, 140/227) of LGGs harbored a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver [@doi:10.1186/s40478-020-00902-z] (**Figure {@fig:Fig2}A**). <!--SAMPLECOUNT-->
+As expected, the majority (62%, 140/226) of LGGs harbored a somatic alteration in _BRAF_, with canonical _BRAF::KIAA1549_ fusions as the major oncogenic driver [@doi:10.1186/s40478-020-00902-z] (**Figure {@fig:Fig2}A**). <!--SAMPLECOUNT-->
 We observed additional mutations in _FGFR1_ (2%), _PIK3CA_ (2%), _KRAS_ (2%), _TP53_ (1%), and _ATRX_ (1%) and fusions in _NTRK2_ (2%), _RAF1_ (2%), _MYB_ (1%), _QKI_ (1%), _ROS1_ (1%), and _FGFR2_ (1%), concordant with previous studies reporting the near universal upregulation of the RAS/MAPK pathway in these tumors resulting from activating mutations and/or oncogenic fusions [@doi:10.1186/s40478-020-00902-z; @doi:10.1016/j.ccell.2020.03.011].
 Indeed, we observed significant upregulation (ANOVA Bonferroni-corrected p < 0.01) of the KRAS signaling pathway in LGGs (**Figure {@fig:Fig5}B**).
 
 #### Embryonal tumors
  <!--SAMPLECOUNT-->
 The majority (N = 95) of embryonal tumors were medulloblastomas that spanned the spectrum of molecular subtypes (WNT, SHH, Group3, and Group 4; see **Molecular Subtyping of CNS Tumors**), as identified by subtype-specific canonical mutations (**Figure {@fig:Fig2}B**).
-We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as other embryonal tumors) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as "Other embryonal tumors" in **Figure {@fig:Fig2}B**) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### High-grade gliomas
 <!--SAMPLECOUNT-->
-Across HGGs, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61), a gene commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
+Across HGGs, we found that _TP53_ (57%, 36/63) and _H3F3A_ (54%, 34/63) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61), a gene commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
```

Comment:
👍  for TP53, H3, but I think ATRX will need a denominator change. Still looks like 7+11 = 18.

Reply:
Good catch, thanks! I see 18/63 as well.

Reply:
Fixed in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/414/commits/52beeebb45b59275ce0994b1316ff76223aaa3e5.